en_US/deploy/cluster/create-cluster.md

```diff
@@ -1,140 +1,142 @@
 
-EMQX's autocluster feature is based on the [Ekka](https://github.com/emqx/ekka) library, the cluster management library developed for Erlang/OTP applications, supporting features like Erlang node service discovery, autocluster, network partition autoheal, and autoclean.
+:::tip
+Manual clustering is not supported in the Core-Replica architecture.
+:::
 
-EMQX supports multiple autocluster strategies:
+### Configure Nodes to Join a Cluster
 
-| Strategy | Description                               |
-| -------- | ----------------------------------------- |
-| manual   | Create a cluster through manual command   |
-| static   | Create a cluster using a static node list |
-| dns      | Create a cluster using DNS A records      |
-| etcd     | Create a cluster via etcd                 |
-| k8s      | Create a cluster via Kubernetes service   |
+After the nodes are started,  you can run the `cluster join` command on the node that you want to join the cluster. For example, you want `emqx@s2.emqx.io` to join  `emqx@s1.emqx.io`, run the command below on `emqx@s2.emqx.io`：
 
-By default, EMQX adopts a manual clustering strategy, which can be set in `emqx.conf`:
+<!--should we add how to start the nodes or how to confirm the nodes are started?-->
 
 ```bash
-cluster {
-    ## options: manual | static | dns | etcd | K8s
-    discovery_strategy  =  manual
-}
+$ ./bin/emqx_ctl cluster join emqx@s1.emqx.io
+
+Join the cluster successfully.
+Cluster status: [{running_nodes,['emqx@s1.emqx.io','emqx@s2.emqx.io']}]
 ```
 
-Note: The mcast discovery strategy has been deprecated and will be removed in future releases.
+:::tip
+
+1. This command must run on the node to join the cluster, that is, as a **request** rather than **invite**.
+
+2. After `emqx@s2.emqx.io` joins `emqx@s1.emqx.io` to form a cluster, it will clear the local data and synchronize the data in `emqx@s1.emqx.io`.
+
+3. If `emqx@s2.emqx.io`  wants to join another cluster, it must first leave the current cluster. On how to leave the cluster, see  [Manage cluster nodes](#manage-cluster-nodes).
+
+   :::
+
+Now you have successfully created a cluster with two nodes, you can read the [Query Cluster Status](#query-cluster-status), [Manage Cluster Nodes](#manage-cluster-nodes), and [Configure Network Protocols](#configure-network-protocols) sections on how to monitor the cluster status and how to manage the cluster. 
+
+## Auto Clustering
+
+Auto clustering in EMQX is another feature that allows multiple EMQX nodes to form a cluster automatically without manual configuration. Auto clustering simplifies the process of setting up an EMQX cluster and makes it easier to add or remove nodes from the cluster dynamically. It also provides fault tolerance and high availability by allowing the cluster to continue operating even if one or more nodes fail.
```

Comment:
> It also provides fault tolerance and high availability by allowing the cluster to continue operating even if one or more nodes fail.

The same applies to the manual cluster as well: it will also keep working if one or several nodes fail, as long as at least one node is still running... And manual cluster will recover automatically as soon as the faulty nodes come up again (given that their data is not wiped); there is no need to manually re-join them to the cluster after the initial command. 

Perhaps this sentence be moved to the top of the chapter, since fault-tolerance is not strictly a property of automatic clustering, but clustering in general.